src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -328,15 +352,19 @@ const ProfilePageContent = ({
               u =>
                 u.circle && (
                   <div key={u.id} className={classes.circle}>
-                    <Avatar
-                      alt={u.circle.name}
-                      src={getCircleAvatar({
+                    <NewApeAvatar
```

Comment:
Hmm, I wonder if we could have a more descriptive name than `NewApeAvatar`

Reply:
Actually, I used the existing `NewApeAvatar` 

It's Intended to replace ApeAvatar. Maybe once we get rid of ApeAvatar, then it will be named accordingly

Reply:
Yeah... Note that @akkikumar72 is just reusing an existing component.

The history here is that "Avatar" itself is a Material-UI component, so when exrhizo added functionality he called the new component ApeAvatar, but it had some weird workarounds for data inconsistency. 

So @teeolendo started a new version of that, and called it NewApeAvatar, but it never got finished. 

Now there's also a second reworking called just "Avatar" again by @amrro in #739 but there's still outstanding feedback on that.

One way forward would be for @akkikumar72 to pull in the new Avatar component from #739, fix the one weird thing about it, and use that in ProfilePage. We could later go through the code and replace instances of ApeAvatar and NewApeAvatar with just the new Avatar, and remove the two old components entirely.

Reply:
Oh, my bad @akkikumar72 didn't realize this historical context.

Reply:
@crabsinger np, I was also not fully sure of its historical context. Thanks, @levity for letting us know.

I have added a small comment for https://github.com/coordinape/coordinape/pull/739, as it's a huge PR and I don't want to pull all of it without it getting approved. 

So, either I will wait, or if it takes time then I will cherry-pick avatar change only.